### ISSUE TITLE
Install on Ubuntu 18.04 instructions did not work.

### ISSUE THREAD

**Issue by MaurGi** [MicrosoftDocs/PowerShell-Docs#5341]
On Ubuntu 18.04 (via WSL 2), following the instructions on this page, I get the following error:

```
The following packages have unmet dependencies:
 powershell : Depends: libicu55 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

Fixed this by running:
```
sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
sudo apt-get update
sudo apt-get install libicu55
```

Courtesy of  this article: [How to download and install libicu55 package for Ubuntu 18.04?](https://askubuntu.com/questions/1066900/how-to-download-and-install-libicu55-package-for-ubuntu-18-04)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37f30869-1dcc-9b77-ecf3-e0ce88bce104
* Version Independent ID: a6f4b56d-aa6c-b557-c0d2-1ff6888c24bd
* Content: [Installing PowerShell Core on Linux - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-7&source=docs)
* Content Source: [reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

**Comment by sdwheeler** [comment]
See #5270

@TravisEz13 This is the second report of problems with this install. What are the correct instructions?

**Comment by TravisEz13** [comment]
My guess is that the 16.04 instructions were followed instead of the 18.04.  I have verify that this does not repro again.

**Comment by TravisEz13** [comment]
the actual 18.04 uses libicu60

**Comment by TravisEz13** [comment]
using the Ubuntu 16.04 package on Ubuntu 18 can result in instability.

**Comment by cawoodm** [comment]
I too followed the 16.04 instructions on 18.04 and then ran the 18.04 instructions. Any way out of this deadend?
```
The following packages have unmet dependencies:
 powershell : Depends: libicu55 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by sdwheeler** [comment]
Please file an issue at https://aka.ms/psfeedback

**Comment by cawoodm** [comment]
Solved:
```
sudo add-apt-repository "deb http://security.ubuntu.com/ubuntu xenial-security main"
sudo apt-get update
sudo apt-get install libicu55
```

**Comment by antman2** [comment]
There's a good chance you first installed `packages-microsoft-prod` by following the instructions for Ubuntu 16.04. That's what I did because those are the instructions at the top of the [Installing PowerShell on Linux](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux) page and I wasn't paying enough attention, i.e.: the wrong instructions:

```bash
$ sudo apt-get update

$ sudo apt-get install -y wget apt-transport-https software-properties-common

$ wget -q https://packages.microsoft.com/config/ubuntu/16.04/packages-microsoft-prod.deb

$ sudo dpkg -i packages-microsoft-prod.deb

$ sudo apt-get update

$ sudo apt-get install -y powershell
...
The following packages have unmet dependencies:
 powershell : Depends: libicu55 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

If you install `libicu55` to "fix" this issue then you'll have broken package dependencies that will cause problems with future updates and upgrades. To fix this, remove the 16.04 version of `packages-microsoft-prod` with:

```bash
$ sudo dpkg --remove packages-microsoft-prod
(Reading database ... 29357 files and directories currently installed.)
Removing packages-microsoft-prod (1.0-2) ...
```

And then resume following the correct instructions for 18.04 (note the different version for `packages-microsoft-prod`):

```bash
$ wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb

$ sudo dpkg -i packages-microsoft-prod.deb
Selecting previously unselected package packages-microsoft-prod.
(Reading database ... 29352 files and directories currently installed.)
Preparing to unpack packages-microsoft-prod-18.04.deb ...
Unpacking packages-microsoft-prod (1.0-ubuntu18.04.2) ...
Setting up packages-microsoft-prod (1.0-ubuntu18.04.2) ...

$ sudo apt-get update

$ sudo add-apt-repository universe

$ sudo apt-get install -y powershell
...
Setting up powershell (7.1.1-1.ubuntu.18.04) ...
Processing triggers for libc-bin (2.27-3ubuntu1.4) ...
Processing triggers for man-db (2.8.3-2ubuntu0.1) ...
```

You should now be able to run PowerShell...

```bash
$ pwsh
PowerShell 7.1.1
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.
```
